python-threatexchange/threatexchange/signal_type/tests/test_raw_text.py

```diff
@@ -8,16 +10,19 @@
 class TestRawTextSignal(MatchesStrAutoTest):
     TYPE = RawTextSignal
 
-    def get_validate_hash_cases(self):
+    @pytest.fixture
+    def validate_hash_cases(self):
         return [
             ("a", "a"),
             ("a ", "a"),
         ]
 
-    def get_compare_hash_cases(self):
+    @pytest.fixture
+    def compare_hash_cases(self):
         return []
 
-    def get_matches_str_cases(self):
+    @pytest.fixture
```

Comment:
blocking: Ditto that this looks like a better match for `pytest.mark.parametrize`

Reply:
Any response on this unaddressed blocking comment?